mmdet/models/backbones/cspnext.py

```diff
@@ -169,6 +169,8 @@ def __init__(
             self.add_module(f'stage{i + 1}', nn.Sequential(*stage))
             self.layers.append(f'stage{i + 1}')
 
+        self._freeze_stages()
```

Comment:
This should not be necessary, as the train() method is called before training to trigger this method

Reply:
I’m not sure why, but it errors out when I don’t add this, see #8208

Reply:
OK, I'll go confirm.